tests/unit/type/zset.tcl

```diff
@@ -2208,6 +2208,16 @@ start_server {tags {"zset"}} {
         assert_match "*syntax*" $err
     }
 
+    test {ZRANGESTORE skiplist} {
+        set original_max [lindex [r config get zset-max-ziplist-entries] 1]
+        r config set zset-max-ziplist-entries 0
+        r flushall
+        r zadd z1{t} 1 a
+        set res [r zrangestore z2{t} z1{t} 0 -1]
+        assert_equal $res 1
+        r config set zset-max-ziplist-entries $original_max
+    }
+
```

Comment:
i would like to change like this:
1. let's mention the test purpose in the title
2. avoid using flushall, we can just del z1{t} z2{t}
3. using `zset-max-listpack-entries` the new alias name
```suggestion
    test {ZRANGESTORE with zset-max-listpack-entries 0 dst key should use skiplist encoding} {
        set original_max [lindex [r config get zset-max-listpack-entries] 1]
        r config set zset-max-ziplist-entries 0
        r del z1{t} z2{t}
        r zadd z1{t} 1 a
        assert_equal 1 [r zrangestore z2{t} z1{t} 0 -1]
        r config set zset-max-ziplist-entries $original_max
    }
```

Reply:
Done